.github/workflows/README.md

```diff
@@ -0,0 +1,134 @@
+# Snyk
+
+We have two Snyk actions. [sbt-node-snyk](sbt-node-snyk.yml) and [sbt-node-snyk-pr](sbt-node-snyk-pr.yml).
+
+It is recommended you use both on your project for greater coverage.
+
+## Requirements
+
+1. `SNYK_TOKEN` must be available in your repo's 'Actions secrets' under 'Organization secrets' (Go to `/settings/secrets/actions`). If you cannot see this, ask an engineering manager to add it to your repo.
+1. `with: ORG:` be sure to edit this in the `yml` examples below. This will be the org ID within Snyk.
+![image](https://user-images.githubusercontent.com/48949546/112194614-f6985880-8c00-11eb-946f-a88fdae57662.jpg)
+
+A convention we have is to have these files called `.github/workflows/snyk.yml` and `.github/workflows/snyk-pr.yml`.
+
+# [`snyk monitor`](https://docs.snyk.io/snyk-cli/commands/monitor)
+
+This action creates a project in your Snyk account to monitor open source SBT and Node vulnerabilities and license issues on a cron job.
+
+## Usage
+
+See [sbt-node-snyk.yml](sbt-node-snyk.yml)
+
+### Basic usage
+
+```yml
+name: Snyk
+
+on:
+  schedule:
+    - cron: '0 6 * * *'
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  security:
+    uses: guardian/.github/.github/workflows/sbt-node-snyk.yml@main
+    with:
+      ORG: ~
+    secrets:
+       SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+```
+
+`ORG` is required and should be your organisation ID in Snyk. If this organisation doesn't exist, it will be created.
```

Comment:
This isn't true and can be seen with [this error](https://github.com/guardian/frontend/runs/6174500338?check_suite_focus=true#step:6:44).

Reply:
```suggestion
`ORG` is required and should be your organisation ID in Snyk.
```